pkg/command/client.go

```diff
@@ -39,12 +37,19 @@ func (c *Client) SyncDashboard(ctx context.Context, uid string, queriesDir strin
 		queriesDirAbs = filepath.Join(wd, queriesDir)
 	}
 
-	queries := map[string]string{}
+	queries := QueryManager{}
```

Comment:
Now that we have a _QueryManager_, it could start hiding some internal details. Eg; resolving the dir absolute path;
```go
func NewQueryManager(dir string) (*QueryManager, error) {
        dirAbs := queriesDir
	if !filepath.IsAbs(dir) {
		wd, err := os.Getwd()
		if err != nil {
			return nil, err
		}
		dirAbs = filepath.Join(wd, dir)
	}
       return &QueryManager{dir: dirAbs}, nil
}

```